packages/react-composites/src/composites/common/Drawer/MoreDrawer.tsx

```diff
@@ -319,26 +340,30 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
         itemKey: 'dynamicSelectionKey',
         text: localeStrings.strings.call.moreButtonGalleryFloatingLocalLayoutLabel,
         onItemClick: () => {
+          /* @conditional-compile-remove(gallery-layout-composite) */
           props.onUserSetGalleryLayout && props.onUserSetGalleryLayout('floatingLocalVideo');
           onLightDismiss();
         },
         iconProps: {
           iconName: 'FloatingLocalVideoGalleryLayout',
           styles: { root: { lineHeight: 0 } }
         },
+        /* @conditional-compile-remove(gallery-layout-composite) */
         secondaryIconProps: props.userSetGalleryLayout === 'floatingLocalVideo' ? { iconName: 'Accept' } : undefined
       },
       {
         itemKey: 'focusedContentSelectionKey',
         text: localeStrings.strings.call.moreButtonGalleryFocusedContentLayoutLabel,
         onItemClick: () => {
+          /* @conditional-compile-remove(gallery-layout-composite) */
```

Comment:
you are removing released features here. this should not be added in any of the places you added it

Reply:
But there are other parts where we have cc ... those were not removed. so it ends up bubbling up errors. For the sake of successful I am adding this. Removing them should be a different KTLO item.